back/engines/free/email_campaigns/app/mailers/email_campaigns/internal_comment_on_idea_assigned_to_you_mailer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module EmailCampaigns
+  class InternalCommentOnIdeaAssignedToYouMailer < ApplicationMailer
+    protected
+
+    def subject
```

Comment:
When we were discussing it, I didn't realize these mailers are 95% the same. I supposed there are more differences.

In this case, creating a base class does make sense, as you originally proposed. Sorry for the confusion.